backend/tests/unit_tests/config/test_create.py

```
@@ -56,14 +57,14 @@ def _run_c_config(
-            with pytest.raises(ValueError) as error:
+            with pytest.raises(APIError) as error:
                 c_config(config_definition_key, config_key, data)
-                mock_execute_query.assert_not_called()
-            assert str(error.value) == expected_error
+            detail = error.value.detail[0]
+            assert detail["msg"] == expected_error
             return
 
         c_config(config_definition_key, config_key, data)
-        mock_execute_query.assert_called_once()
+        mock_execute_query.call_count == 2
```

Ruff (0.8.2)

Pointless comparison at end of function scope. Did you mean to return the expression result?

(B015)